reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,64 @@
+  - `mount_type` set to `efs`
+  - `nfs_extra_options` set to `tls`
+* The Anbox Application Registry(AAR) can make use of Identity and Access Management(IAM) roles applied to an AWS instance using instance profiles. This relieves you from having to configure an access key/secret for instances. For information on how to use an IAM role in instance profiles, see [AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html). <!--AC-2025/1700-->
+* The NATS charm is switched from its [older version](https://charmhub.io/nats-charmers-nats) to a [newer version](https://charmhub.io/nats) on Charmhub. This would require that you switch to the new charm source. For more information, see [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/how-to-upgrade-anbox-cloud/17750).
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Launching an instance independent of a session is now possible using the Anbox Cloud dashboard.  This is especially useful for situations where you want to create an instance but do not want to stream anything. To do this, select the *Create Instance* button on the *Instances* view, provide the information required for instance creation and select *Create and Start*.
+* Support multiple subclusters from the dashboard for large scale deployments.
+* Client side upscaling is now available as an experimental feature.You can now add `upscaling` as a query parameter to your session URL. For example, `http://<ip_address_of_instance>/sessions/<session_ID>?upscaling=true`.
+* Experimental support for managing the [Vehicle Hardware Abstraction Layer (VHAL)](https://source.android.com/docs/automotive/vhal) properties for [AAOS](https://source.android.com/docs/automotive/start/what_automotive) images.
+* Enhancements for improved responsiveness, error handling and messages
+* Updated Python and Javascript dependencies.
+
+### Other
+
+* Android security updates for February 2024 (see [Android Security Bulletin - February 2024](https://source.android.com/docs/security/bulletin/2024-02-01) for more information).<!--AC-2249-->
+* The Android WebView has been updated to [121.0.6167.143](https://chromereleases.googleblog.com/2024/01/chrome-for-android-update_0251787784.html).
+
+## Removed functionality
+
+* Clustering support (which existed as an experimental feature) for Anbox Cloud Appliance is removed.<!--AC-2176-->
+* The deprecated Android 11 images are now unsupported and hence removed.<!--AC-2108-->
+
+## Known issues
+
+* In the Anbox Cloud dashboard, selecting the *Download all* button to download all logs from an instance in *Error* state displays a *Page not found (404)* error.<!--AC-2284-->
```

Comment:
This should have a public bug.

Reply:
Created one - [LP 2053106](https://bugs.launchpad.net/anbox-cloud/+bug/2053106)